.env.sample

```diff
@@ -0,0 +1,16 @@
+# rename to .env; turn on Less secure app access in gmail setting!!
+
+MAIL_USERNAME = sample@gmail.com
+MAIL_FROM = sample@gmail.com
+MAIL_PASSWORD = secret_password
```

Comment:
**Issue**: Storing sensitive information such as 'MAIL_PASSWORD' in plaintext within a version-controlled file poses a significant security risk. <br> **Fix**: Use environment variables or encrypted secrets management solutions to handle sensitive information securely. <br> **Code Suggestion**: 
 ```
 -MAIL_PASSWORD = secret_password
 +MAIL_PASSWORD = env('MAIL_PASSWORD')
 ```